modules/schema/src/index.js

```diff
@@ -71,5 +74,5 @@ export default ({
     });
   }
 
-  return schema;
+  return middleware?.length ? applyMiddleware(schema, ...middleware) : schema;
```

Comment:
tiny thing but does it fail with an empty array? if not we can probably remove this check, `middleware` is defaulted to an array anyway

Reply:
lol you're right, just tested w/ empty array, will update